.travis.yml

```diff
@@ -1,3 +1,7 @@
+dist: trusty
+
+sudo: required
```

Comment:
Why?

Reply:
actually agreed, for the current configuration it is not reuquired.